test/unit/flow_presenter_test.rb

```diff
@@ -141,4 +141,12 @@ def flow_registry
   test "#start_page_link returns the start page link for a non response store flow" do
     assert_equal "/flow-name/y", @flow_presenter.start_page_link({ "key" => "value" })
   end
+
+  test "#start_page_link returns the new flow when we're in marriage-abroad" do
+    @flow = SmartAnswer::Flow.build do
+      name "marriage-abroad"
+    end
+    @flow_presenter = FlowPresenter.new(@flow, nil)
+    assert_equal "https://www.prove-eligibility-foreign-government.service.gov.uk/before-you-start/which-country-are-you-getting-married-in", @flow_presenter.start_page_link({ "key" => "value" })
```

Comment:
Real nitpicky, but could we format this assertion onto multiple lines to be a bit more readable (like some of the `#change_answer_link` tests)?